### DISCUSSION TITLE
v1.9.*

### DISCUSSION THREAD

**Discussion by ckken** [empjs/emp#158] [Ideas]
## 新功能 
+ 增加**多入口模式** 参考next js 的 `pages`方式扫描生成多页面 `完成` by `Ken`
+ 增加 **Library 模式** 支持包开发 并支持 `umd` `esm` `emp` 等共享方式  `进行中` by `Ken`

## 修复 
+ **跨站热**更问题 [issue](https://github.com/efoxTeam/emp/issues/8) `完成` by `Ken`

## 愿景
+ 增加官方文档支持
+ 支持 SSR 去中心化渲染服务 
+ 支持 SSG 生成并一键上云 
+ 在线 STORYBOOK 支持

**Comment by ckken** [comment]
+ 插件化 SSR 参考 https://vitejs.dev/guide/ssr.html
+ 插件化 SSG 参考 https://vitejs.dev/guide/ssr.html#pre-rendering-ssg

**Reply by ckken** [reply]
https://github.com/efoxTeam/emp/tree/v1.9.x 版本分支

**Reply by ckken** [reply]
1 清楚 react 相关内容 整合 到 @efox/emp-react
2 清楚弃用文件

**Reply by ckken** [reply]
自动多入口文档 https://github.com/efoxTeam/emp/wiki/v1.8.25-%E8%87%AA%E5%8A%A8%E5%A4%9A%E5%85%A5%E5%8F%A3%E6%94%AF%E6%8C%81

**Reply by ckken** [reply]
考虑是否加入 disable eslint 选项 如 https://github.com/facebook/create-react-app/blob/bb64e31a81eb12d688c14713dce812143688750a/packages/react-scripts/config/webpack.config.js#L59

**Comment by doerme** [comment]
1.添加特定后缀产物压缩配置，以及压缩率配置。
2.添加dataUrl转换最小体积配置。

**Comment by lilililee** [comment]
1. 优化开发模式更新编译缓慢；
2. 调整 devtool 配置。
